### ISSUE TITLE
Exception if object positions have fractional coordinates

### ISSUE THREAD

**Issue by danielscherzer** [Ragath/TiledLib.Net#6]
I use the nuget package version 2.3.0
I found an error when playing around with object layers:

Steps to reproduce the error: 
In Tiled place a point in an object layer on a fractional coordinate, like `X=550.34, Y=500.67`. 
When loading the tmx file with your library I get an exception: 
```
System.InvalidOperationException
  HResult=0x80131509
  Message=There is an error in XML document (98, 4).
  Source=System.Private.Xml
  StackTrace:
   at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   at System.Xml.Serialization.XmlSerializer.Deserialize(TextReader textReader)
   at TiledLib.Utils.ReadTmxMap(TextReader reader)
   at TiledLib.Map.FromStream(Stream stream, Func`2 tsLoader)
   at Example.Map..ctor(Stream stream, Func`2 loadDependentNamedStream) in ...

Inner Exception 1:
InvalidOperationException: There is an error in XML document (98, 4).

Inner Exception 2:
FormatException: Input string was not in a correct format.
```
If I change the position to an integer coordinate everything works fine.

Cheers,
Daniel

**Comment by Ragath** [comment]
Resolved in v3.0.0

**Comment by danielscherzer** [comment]
Thanks for the quick response! I Just tried out nuget version 3.0:
Now data types of the x,y,width and height fields are double, but strangly enough the same exception is still thrown, if one of the fields is a fractional coordinate. I also did the change directly in the tmx file:
```
  <object id="4" name="player" gid="188" x="861" y="744" width="32" height="32"/>
```
works
```
  <object id="4" name="player" gid="188" x="861.5" y="744" width="32" height="32"/>
```
does not, which of course is the same thing Tiled does. 
I work on a German machine, so I also tried 861,5 to eliminate language local trouble.

Cheers,
Daniel
